<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrokerDialog</class>
 <widget class="QDialog" name="BrokerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Connections</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="layout_2">
       <item>
        <widget class="QListView" name="brokers"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout">
         <item>
          <widget class="QPushButton" name="add">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="profileNameLabel">
           <property name="text">
            <string>Profile Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="profileName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="brokerAddressLabel">
           <property name="text">
            <string>Broker Address</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="brokerAddress"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="brokerPortLabel">
           <property name="text">
            <string>Broker Port</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="brokerPort"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="clientIdLabel">
           <property name="text">
            <string>Client ID</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="clientId"/>
            </item>
            <item>
             <widget class="QPushButton" name="generate">
              <property name="text">
               <string>Generate</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QTabWidget" name="tabWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="general">
            <attribute name="title">
             <string>General</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="keepAliveLabel">
               <property name="text">
                <string>Keep Alive Interval</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="keepAlive"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="cleanSessionLabel">
               <property name="text">
                <string>Clean session</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="cleanSession">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="autoReconnectLabel">
               <property name="text">
                <string>Auto Reconnect</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="autoReconnect">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="mqttVersionLabel">
               <property name="text">
                <string>MQTT Version</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="mqttVersion"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="userCredentials">
            <attribute name="title">
             <string>User Credentials</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="1">
              <widget class="QLineEdit" name="username"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="usernameLabel">
               <property name="text">
                <string>User Name</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="passwordLabel">
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="password"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="ssl">
            <attribute name="title">
             <string>SSL/TLS</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout">
             <item row="1" column="1">
              <widget class="QComboBox" name="sslProtocol"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="sslProtocolLabel">
               <property name="text">
                <string>Protocol</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="enableSsl">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="enableSslLabel">
               <property name="text">
                <string>Enable SSL/TLS</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ok">
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="apply">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
